llvm/lib/CodeGen/GlobalISel/CallLowering.cpp

```diff
@@ -751,6 +751,7 @@ bool CallLowering::handleAssignments(ValueHandler &Handler,
     const LLT NewLLT = Handler.isIncomingArgumentHandler() ? LocTy : ValTy;
     const EVT OrigVT = EVT::getEVT(Args[i].Ty);
     const LLT OrigTy = getLLTForType(*Args[i].Ty, DL);
+    const LLT PointerTy = LLT::pointer(0, DL.getPointerSizeInBits(0));
 
     // Expected to be multiple regs for a single incoming arg.
```

Comment:
> // Expected to be multiple regs for a single incoming arg.

This is not true in the case of indirect assignment. Should we update the comment so the comment remains accurate? Alternatively, should `NumParts` be 1 when there is indirect assignment since we expect only one reg for an indirect assignment?

Reply:
Well it is still true I think. The `deterimenAssignments` callback still creates four assignment with four registers with four flags (I am talking about the example that is rv32 with 128 bit registers), but those regs wont be actually created in the GMIR since here we handle indirect parameter passing. I think it may be still usefule information, that if the value were to be split into different register how would it fit in them (basically that is the infor created by `deterimenAssignments` if I am correct). Nevertheless I still not came up with an idea how to use that extra information but who knows when will it be good :) .